plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveFaultTolerantExecutionConnectorTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive;
+
+import io.trino.testing.BaseFaultTolerantExecutionConnectorTest;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class TestHiveFaultTolerantExecutionConnectorTest
```

Comment:
I often run into "too many files opened" error when running this test locally. @arhimondr, do you think this is an issue that we need to resolve?

Reply:
In Trino deploymen guidelines we suggest quite big limits so maybe this is not a big deal when run on real cluster ( https://trino.io/docs/current/installation/deployment.html ). What is the limit on your local env where you get the errors?

Reply:
Default is 256. Now I have changed to 10240 and it worked.

I understand this is not an issue in production environment, but it may cause confusion to developers?